schema/CbsdRecordData.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Registration parameters provided for a CBSD",
+  "type": "object",
+  "required": [ "fccId", "cbsdCategory", "airInterface", "installationParam", "measCapability"],
```

Comment:
"userId" should be required.

Reply:
I don't think they are required for CBSD dump record, because they are SAS credential information.

Reply:
Ok.